setup.py

```diff
@@ -379,6 +437,7 @@ def get_common_options(build_ext):
     cpp_flags = get_cpp_flags(build_ext)
     link_flags = get_link_flags(build_ext)
     mpi_flags = get_mpi_flags()
+    mxnet_include_dirs = os.environ.get('INCLUDES')
```

Comment:
I'm not sure this environment variable will be set often.  Is there any other way?

Reply:
We will update this soon once the include path is added to MXNet pip package.

Reply:
Use an API to get the path now.